Guardfile

```diff
@@ -16,10 +18,10 @@
 # and, you'll have to watch "config/Guardfile" instead of "Guardfile"
 
 guard 'reek' do
-  watch(%r{.+\.rb$})
+  watch(/.+\.rb$/)
   watch('.reek')
 end
 
-guard :rspec, cmd: "bundle exec rspec" do
+guard :rspec, cmd: 'bundle exec rspec' do
   watch(%r{^spec/.+_spec\.rb$})
 end
```

Comment:
Rubocop won't run when entering `guard`.

There's a command that can add rubocop here https://github.com/rubocop/guard-rubocop